groups/rds/rds.tf

```diff
@@ -0,0 +1,130 @@
+# ------------------------------------------------------------------------------
+# RDS Security Group and rules
+# ------------------------------------------------------------------------------
+module "rds_security_group" {
+
+  source  = "terraform-aws-modules/security-group/aws"
+  version = "~> 3.0"
+
+  name        = "sgr-smartview-rds-001"
+  description = "Security group for the smartview RDS database"
+  vpc_id      = data.aws_vpc.vpc.id
+
+  ingress_cidr_blocks = concat(local.admin_cidrs, var.rds_onpremise_access)
+  ingress_rules       = ["oracle-db-tcp"]
+  ingress_with_cidr_blocks = [
+    {
+      from_port   = 5500
+      to_port     = 5500
+      protocol    = "tcp"
+      description = "Oracle Enterprise Manager"
+      cidr_blocks = join(",", concat(local.admin_cidrs, var.rds_onpremise_access))
+    }
+  ]
+  ingress_with_source_security_group_id = []
+
+  egress_rules = ["all-all"]
+}
+
+# ------------------------------------------------------------------------------
+# RDS Instance
+# ------------------------------------------------------------------------------
+module "smartview_rds" {
+  source  = "terraform-aws-modules/rds/aws"
+  version = "2.23.0" # Pinned version to ensure updates are a choice, can be upgraded if new features are available and required.
+
+  create_db_parameter_group = "true"
+  create_db_subnet_group    = "true"
+
+  identifier                 = join("-", ["rds", var.identifier, var.environment, "001"])
+  engine                     = "oracle-se2"
+  major_engine_version       = var.major_engine_version
+  engine_version             = var.engine_version
+  auto_minor_version_upgrade = var.auto_minor_version_upgrade
+  license_model              = var.license_model
+  instance_class             = var.instance_class
+  allocated_storage          = var.allocated_storage
+  storage_type               = var.storage_type
+  iops                       = var.iops
+  multi_az                   = var.multi_az
+  storage_encrypted          = true
+  kms_key_id                 = data.aws_kms_key.rds.arn
+
+  name     = upper(var.name)
+  username = local.smartview_rds_data["admin-username"]
+  password = local.smartview_rds_data["admin-password"]
+  port     = "1521"
+
+  deletion_protection       = true
+  maintenance_window        = var.rds_maintenance_window
+  backup_window             = var.rds_backup_window
+  backup_retention_period   = var.backup_retention_period
+  skip_final_snapshot       = "false"
```

Comment:
Could remove the quotes here to be consistent with others.

Reply:
Same is true in some other lines too - noticed some of the "trues" are in quotes and some aren't.

Reply:
Updated in latest commits